doc/phpactor.txt

```diff
@@ -191,20 +191,22 @@ Phpactor does not assume any mappings automatically, the following mappings
 are available for you to copy:
 >
 
-  nmap <Leader>u :PhpactorImportClass<CR>
-  nmap <Leader>ua :PhpactorImportAllClasses<CR>
-  nmap <Leader>mm :PhpactorContextMenu()<CR>
-  nmap <Leader>nn :PhpactorNavigate()<CR>
-  nmap <Leader>oo :PhpactorGotoDefinition()<CR>
-  nmap <Leader>oh :PhpactorGotoDefinitionHsplit()<CR>
-  nmap <Leader>ov :PhpactorGotoDefinitionVsplit()<CR>
-  nmap <Leader>ot :PhpactorGotoDefinitionTab()<CR>
-  nmap <Leader>K :PhpactorHover()<CR>
-  nmap <Leader>tt :PhpactorTransform()<CR>
-  nmap <Leader>cc :PhpactorClassNew()<CR>
-  nmap <silent><Leader>ee :PhpactorExtractExpression(v:false)<CR>
-  vmap <silent><Leader>ee :<C-U>PhpactorExtractExpression(v:true)<CR>
-  vmap <silent><Leader>em :<C-U>PhpactorExtractMethod()<CR>
+  augroup PhpactorMappings
```

Comment:
Like I said this should not be needed if the mappings are properly created.
Instead what about explaining that the mappings should be added as a ftplugin ?

Reply:
But this is for copy-pasting into your `.vimrc` - at least that's how I see them,

Reply:
The file `ftplugin/php/mappings.vim` so it's not obvious to me.
Also it's a lot better practice to not put it in the vimrc.
In this case we don't want these mappings when editing other files than PHP.

Reply:
The goal of not including mappings into phpactor was to let the user do it.
But we could still tell them how to do it properly.

If you put those mappings into your `vimrc` you will ends up having the mappings delcared for any other filetype.
Which will bring error since the plugin is now in the `ftplugin` directory the mapping will exists but not the commands.
Also by doing so we increase drastically the chances of conflict with another plugin mappings or even user defined mappings.

Reply:
I think this is outside the scope of this PR. This is just fixing the example mappings in the help.

Reply:
I would still avoid deciding on some default mappings and just suggest them.

People will come from different tools, have different shortcuts, and the defaults might conflict. I would rather people just have the ability to choose and update their mappings themselves.

Reply:
> I would still avoid deciding on some default mappings and just suggest them.

I know and I don't suggest to do so here.

> I think this is outside the scope of this PR. This is just fixing the example mappings in the help.

This PR changes the way the doc explains how to declare mappings.
I'm only suggesting to have only one `autocmd` that will call a function that will declare the mappings just as they are right now.

If it's still unclear feel free to resolve, no need to lost too much time on something that is commented anyway :)